tests/secrets.test.js

```diff
@@ -8,7 +8,7 @@ const read = (file) => fs.readFileSync(path.resolve(file), { encoding: "utf8" })
 // eslint-disable-next-line security/detect-non-literal-fs-filename
 const rm = (...files) => files.forEach((file) => fs.unlinkSync(path.resolve(file)));
 
-const PROCESS_ENV = process.env;
+const PROCESS_ENV = Object.assign({}, process.env);
```

Comment:
optional, just fyi: the newer, cleaner way of doing this is by using the spread operator

```suggestion
const PROCESS_ENV = { ...process.env };
```